core/domain/suggestion_services_test.py

```diff
@@ -2084,6 +2094,1155 @@ def test_create_translation_contribution_stats_from_model(self) -> None:
+            'Accepted')
+        suggestion_services.update_translation_review_stats(
+            suggestion_services.get_suggestion_by_id(
+                initial_suggestion.suggestion_id)
+        )
+        suggestion_services.update_translation_review_stats(
+            suggestion_services.get_suggestion_by_id(
+                latest_suggestion.suggestion_id)
+        )
+        translation_review_stats_model = (
+            suggestion_models.TranslationReviewStatsModel.get(
+                'hi', self.reviewer_id, '0'
+            )
+        )
+        translation_contribution_stats_model = (
+            suggestion_models.TranslationContributionStatsModel.get(
+                'hi', self.author_id, '0'
+            )
+        )
+        # Assert translation review stats after the review.
+        # At this point we can confirm that there should be an associated
+        # translation review stat object for the given IDs since we have
+        # called update_translation_review_stats function to create/update
+        # translation review stats.
+        assert translation_review_stats_model is not None
+        self.assertEqual(
+            translation_review_stats_model.accepted_translations_count,
+            2
+        )
+        self.assertEqual(
+            translation_review_stats_model.accepted_translation_word_count,
+            4
+        )
+        self.assertEqual(
+            (
+                translation_review_stats_model
+                .reviewed_translation_word_count
+            ),
+            4
+        )
+        self.assertEqual(
+            translation_review_stats_model
+            .accepted_translations_with_reviewer_edits_count,
+            1
+        )
+        # Assert translation contribution stats after the review.
+        # At this point we can confirm that there should be an associated
+        # translation contribution stat object for the given IDs since we have
+        # called update_translation_contribution_stats_at_submission function
+        # to create/update translation contribution stats.
+        assert translation_contribution_stats_model is not None
+        self.assertEqual(
+            translation_contribution_stats_model.submitted_translations_count,
+            2
+        )
+        self.assertEqual(
+            (
+                translation_contribution_stats_model
+                .submitted_translation_word_count
+            ),
+            6
+        )
+        self.assertEqual(
+            translation_contribution_stats_model.accepted_translations_count,
+            2
+        )
+        self.assertEqual(
+            (
+                translation_contribution_stats_model
+                .accepted_translations_without_reviewer_edits_count
+            ),
+            1
+        )
+
+    def _create_question_suggestion(
+        self,
+        skill_id: str
+    ) -> suggestion_registry.SuggestionAddQuestion:
+        """Creates a topic for a question.
```

Comment:
That's not what this function does, right?

```suggestion
        """Creates a question suggestion corresponding to the supplied skill.
```

Reply:
Done